### PR TITLE
#to solve issue11

### PR DESCRIPTION
**Pull request by ltyDion**
Remove occurrence of PopUpSample in Router.jsx and fix the linter error in PopUpSample.jsx
I do not see the use PopUpSample.jsx in any other files, though I did not delete it. However, in order to pass the linter, I just add the role as "presentation" because I do not know which role they belong to.

### CODE REVIEW

**Comment by Liby99** [comment]
fix #11

File: src/components/PopUpSample.jsx [hunk 1]
```diff
@@ -19,16 +15,17 @@ class PopUpSample extends React.Component {
   }
 
   toggle() {
-    if (this.state.opened) {
-      console.log('close');
+    const { opened } = this.state;
+
+    if (opened) {
       this.setState({ opened: false });
```
Comment (Liby99) [inline src/components/PopUpSample.jsx]: Change this and the following few lines to 
```
this.setState({ opened: !opened });
```
To remove duplicated code

**Comment by Liby99** [comment]
btw, in the future, add `fix #..` in your pull request description to refer to the corresponding issue ticket.

File: src/components/PopUpSample.jsx (hunk 1 above)
Comment (ltyDion) [reply src/components/PopUpSample.jsx]: This sample pop up is written by liby! Zhe duan daima wo kan mengbi le!

**Pull request merged**